dataprofiler/tests/data_readers/test_csv_graph_data.py

```diff
@@ -171,7 +180,7 @@ def test_data_loader_nodes(self):
             options = dict()
             if not GraphData.is_match(input_file["path"], options):
                 return
-            data = GraphData(input_file["path"], options)
+            data = GraphData(input_file["path"], None, options)
```

Comment:
might notate this as `data=None` just so folks know when they are reading this what that param is

Reply:
`input_file_path=input_file["path"]), data=None, options`